core/modules/user/user.admin.inc

```diff
@@ -550,29 +550,37 @@ function user_admin_roles($form, $form_state) {
     '#markup' => t('Roles provide a way to organize people into groups that have similar duties. Each user account can be given one or more roles, and <a href="@permissions">permissions</a> can be granted to those roles. To make the <a href="@permissions">permissions</a> page easier to navigate, it is recommended that roles be ordered from the least permissive (Anonymous) to the most permissive (usually Administrator).', array('@permissions' => url('admin/config/people/permissions'))),
   );
 
-  // Administrative role option.
-  $form['admin_role'] = array(
+  // Default administrator role and content editor role options.
+  $form['default_roles'] = array(
     '#type' => 'fieldset',
-    '#title' => t('Administrator role'),
+    '#title' => t('Default roles'),
+    '#description' => t('Changing these settings will not affect existing permissions.'),
     '#collapsible' => TRUE,
     '#collapsed' => TRUE,
     '#weight' => -20,
   );
   $roles = user_roles();
-  // Do not allow the anonymous or authenticated roles to be set as the
-  // administrator.
+  // Do not allow the anonymous or authenticated roles to be set as the default
+  // administrator or content editor roles.
   unset($roles[BACKDROP_ANONYMOUS_ROLE]);
   unset($roles[BACKDROP_AUTHENTICATED_ROLE]);
   // Add the "disabled" option to the top of the roles list (makes sure that
-  // this is the selected option if the role that was previously set as admin is
-  // deleted).
+  // this is the selected option if the role that was previously set as admin or
+  // content editor is deleted).
   $roles = array(0 => t('disabled')) + $roles;
```

Comment:
I realize this is an existing string, but I think we should change this to either `- None-` to match other disabled options, or if we want to keep this word, change to `- Disabled -`.

```suggestion
  $roles = array(0 => t('- None -')) + $roles;
```